js/phyx.js

```diff
@@ -1026,7 +1047,7 @@ class PhylorefWrapper {
     // Set the @id and @type.
     phylorefAsJSONLD['@id'] = phylorefURI;
     phylorefAsJSONLD['@type'] = [
-      'http://phyloinformatics.net/phyloref.owl#Phyloreference',
+      'phyloref:Phyloreference',
       'owl:Class',
```

Comment:
Isn't `owl:Class` guaranteed to be redundant?

Reply:
It isn't redundant because of `phyloref:Phyloreference` -- without that `owl:Class`, the reasoner will assume that this URI refers to an individual belonging to `phyloref:Phyloreference` rather than a class in its own right. However, it *is* implied by the domain of the `owl:equivalentClass` statement we make (https://github.com/phyloref/curation-tool/blob/3fa2449deeb00fb17ee2745a92438198400ac74d/js/phyx.js#L1160). I think making it explicit here is clearer than letting it be inferred later on, but you're right, this is otherwise redundant and can be removed. What do you think?

Reply:
I guess we can keep it. But can you add a comment making it clear why it's not entirely redundant, so someone coming across this in the future won't ask the question again.

Reply:
Done in fcfd75f!